root/entity/Details.js

```diff
@@ -16,28 +16,37 @@ import EntityLink from '../static/scripts/common/components/EntityLink';
 import chooseLayoutComponent from '../utility/chooseLayoutComponent';
 import formatUserDate from '../utility/formatUserDate';
 
-type Props = {
+type WSLinkProps = {
+  +entityGid: string,
+  +entityProperties: {[property: string]: any},
```

Comment:
We don't have a flow type for this (ENTITIES[type]) yet, right? Should we have a more detailed one?